tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaApisTest.java

```diff
@@ -803,7 +812,89 @@ public void testBrokerRespectsPartitionsOrderAndSizeLimits() throws Exception {
             maxPartitionBytes, maxResponseBytes, messagesPerPartition);
     }
 
+    @DataProvider(name = "allowAutoTopicCreation")
+    public static Object[][] allowAutoTopicCreation() {
+        return new Object[][]{
+                { true, true, true },
+                { true, true, false},
+                { true, false, true },
+                { true, false, false},
+                { true, null, true },
+                { true, null, false},
+                { false, true, true },
+                { false, true, false},
+                { false, false, true },
+                { false, false, false},
+                { false, null, true },
+                { false, null, false}
```

Comment:
```suggestion
                { true, true, true },
                { true, true, false },
                { true, false, true },
                { true, false, false },
                { true, null, true },
                { true, null, false},
                { false, true, true },
                { false, true, false },
                { false, false, true },
                { false, false, false },
                { false, null, true },
                { false, null, false }
```

Keep the same code style.